rs/https_outcalls/adapter/Cargo.toml

```diff
@@ -26,13 +26,18 @@ serde_json = { workspace = true }
 slog = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true }
+tokio-rustls = { workspace = true }
```

Comment:
should not this be part of the dev deps ?

Reply:
Yes, good spot.